features/opennms-es-rest/src/main/java/org/opennms/plugins/elasticsearch/rest/EventToIndex.java

```diff
@@ -42,9 +42,11 @@
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
 
 import javax.xml.bind.DatatypeConverter;
 
+import org.json.JSONArray;
```

Comment:
I found that if you use org.json.JSONArray, it will insert a name tag for the array in the JSON structure. You can correct this issue by using org.json.simple.JSONArray instead and using .add instead of .put for adding members.

Reply:
Yeah, but I am doing a `body.getAsJsonString()` instead of `body`, which solves the issue as well, but I am in favor of not using 2 different packages. Will change to `org.json.simple`. Thanks for the comment.